appcelerator-archive/amp
# TICK images 1.1 RC

- TICK images update to RC of influxdata 1.1

how to test:
- swarm pull
- swarm start
- `amp --server=localhost:8080 stats` should display stats from the services (CLI tests are failing for other reason at time of this writing)
- connect to localhost:6001 and check that the realtime dashboard in grafana is fed with data, filter by service to validate the change
- make test
